examples/routing_service/shmem_udp_gateway/variables.bat

```diff
@@ -0,0 +1,17 @@
+REM Multicast example
+
+set NDDSHOME="C:/Program Files/rti_connext_dds-7.2.0"
```

Comment:
Like I mentioned before, this file would no longer be necessary. Also, this assumes you're using that installation directory and version 7.2.0, which is not always going to be the case.

Reply:
@fgaranda I could change the README file and the environment variables script to say something like: "Feel free to change the values of the environment variables and adapt them to your system." Would that be okay with you?